packages/contracts-bedrock/contracts/universal/ProxyAdmin.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import { Proxy } from "./Proxy.sol";
+import { Owned } from "@rari-capital/solmate/src/auth/Owned.sol";
+
+/**
+ * @title ProxyAdmin
+ * @dev This is an auxiliary contract meant to be assigned as the admin of a
+        Proxy, based on the OpenZeppelin implementation.
+ */
+contract ProxyAdmin is Owned {
```

Comment:
Can we have an "isUpgrading" function here so we can finally resolve the ChugSplashProxy issues

Reply:
Does this contract work with chugsplash though? I thought the interface of the proxy changed.

Reply:
We're modifying this to be compatible